generic/tests/module_probe.py

```diff
@@ -12,6 +12,8 @@ def run(test, params, env):
 
     This tests the kernel pre-installed kernel modules
     """
+    vm = env.get_vm(params["main_vm"])
```

Comment:
please use env.get_all_vms() to get VM object from env and destroy it before startup your test